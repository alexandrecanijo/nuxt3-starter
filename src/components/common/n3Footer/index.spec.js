import { mountSuspended } from '@nuxt/test-utils/runtime';
import testTarget from './index.vue';

it('can mount the component', async () => {
  const wrapper = await mountSuspended(testTarget);
  expect(wrapper.html()).toMatchSnapshot();
});
