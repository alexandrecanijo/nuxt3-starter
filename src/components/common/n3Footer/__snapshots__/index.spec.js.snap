// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`can mount the component 1`] = `
"<footer data-v-349c2468="">
  <div data-v-349c2468="" class="wrapper">
    <div data-v-349c2468="" class="inner-wrapper"><span data-v-349c2468="">Footer</span></div>
  </div>
</footer>"
`;
